src/tests/ftest/rebuild/cascading_failures.yaml

```diff
@@ -12,7 +12,9 @@ timeout: 360
 server_config:
   name: daos_server
   servers:
-    targets: 8
+#   skipForTicket DAOS-2799
```

Comment:
So we want to avoid using comment to disable a test in yaml.
I'm thinking here that we should use mux for server target so avocado treats it a a variant.
Then in python use cancelForTicket/skipForTicket decorator to disable it.
So the person fixing the issue will just have to remove the decorator to re-enable.
Make sense?

Reply:
Since @ skipForTicket is not implemented in test.yaml, have to use "#" in yaml.
In rebuild-test, I ran tests with !mux with targets 1 and 8. Unlike the other tests, rebuild failure left the server in the failed/unknown state and causing the rest tests failed. We should avoid to use "!mux" in rebuild test.

Reply:
I have added your comment to DAOS-6364, if it is needed to have separate tests for rebuild cascading target:1 and target:8.

Reply:
I cannot land this because we cannot skip test using hash to comment in yaml.
Can we define another function in the .py where one function runs with target=1 and the other runs with target=2, then use skipForTicket on the one with target=8?

Reply:
Thinking more about it, i think you can also leave the target mux in yaml. Then, use cancelForTicket to cancel the test when target=8.
Example of cancelForTicket can be found in 
daos/src/tests/ftest/rebuild/container_create.py